README.md

```diff
@@ -23,7 +23,6 @@ $ composer require rubix/ml
 
 #### Optional
 
-- [Extras Package](https://github.com/RubixML/Extras) for experimental features
```

Comment:
Why remove?

Reply:
We've moved most of the good stuff out of Extras and into the main repo.